git_stacktrace/server.py

```diff
@@ -0,0 +1,195 @@
+from __future__ import print_function
+
+import logging
+import os
+import simplejson as json
```

Comment:
Any reason to use `simplejson` here instead of the stdlib `json` module? I doesn't look like you need any simplejson-specific behavior, and using the stdlib module would let you drop the new dependency.